packages/@webex/plugin-meetings/src/meeting/request.ts

```diff
@@ -260,8 +262,15 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       };
     }
 
-    if (roapMessage) {
-      body.localMedias = roapMessage.localMedias;
+    if (roapMessage || reachability) {
+      body.localMedias = [
+        {
+          localSdp: JSON.stringify({
```

Comment:
why do we include reachability in this now and before we didn't?

Reply:
because previously we never included any roap message with the join request, now we include the TURN_DISCOVERY_REQUEST Roap message and that requires reachability to be included in the request.